Assets/Scripts/Strategy/GameResources/GoldSubscriber.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SwordAndBored.Strategy.GameResources
+{
+    public class GoldSubscriber : MonoBehaviour, IResourceSubscriber
+    {
+        public Gold resource;
```

Comment:
Use the `AssertHelper.IsSetInEditor` function to make sure these are set from the editor. See this https://github.com/wynn-rj/sword-and-bored-game/wiki/_new#asserthelperissetineditor for help in doing that.

Reply:
Will add this in next timebox